packages/fast-components-react-msft/src/utilities/index.ts

```diff
@@ -0,0 +1,4 @@
+const DisplayNamePrefix: string = "Msft";
```

Comment:
Do we want `Msft` or `MSFT`? (the later seems more correct to me)

Reply:
MSFT seems best here IMO.

Reply:
MSFT it is